docs/.vuepress/config.js

```diff
@@ -19,27 +19,38 @@ module.exports = {
         editLinks: true, //  "Edit this page" at the bottom of each page
         lastUpdated: 'Last Updated', //  "Last Updated" at the bottom of each page
         repo: 'nuwave/lighthouse', //  Github repo
-        docsDir: 'docs/', //  Github repo docs folder    
+        docsDir: 'docs/', //  Github repo docs folder
         versions: {
             latest: versioning.versions.latest,
             selected: versioning.versions.latest,
             all: versioning.versions.all
         },
-        search: false,
         nav: [
             {
-                text: `Docs`,
+                text: 'Docs',
                 items: versioning.linksFor('getting-started/installation.md') // TODO create custom component
             },
+            {
+                text: 'Tutorial',
+                link: '/tutorial/'
+            },
             {
                 text: 'Resources',
-                link: '/pages/resources.md'
+                link: '/resources/'
             },
             {
                 text: 'Users',
-                link: '/pages/users.md'
+                link: '/users/'
             }
         ],
         sidebar: versioning.sidebars
-    }
+    },
+    plugins: [
+        ['@vuepress/back-to-top', true],
+        ['@vuepress/medium-zoom', true],
+        ['@vuepress/search', {
```

Comment:
Yay ❤️

Reply:
I just added one more minor change so that the latest stable version is searched by default